receiver/mongodbatlasreceiver/README.md

```diff
@@ -62,8 +62,17 @@ MongoDB Atlas [Documentation](https://www.mongodb.com/docs/atlas/reference/api/l
   - `enabled` (default false)
   - `projects` (required if enabled)
     - `name` (required if enabled)
+    - `collect_host_logs` (default true)
     - `collect_audit_logs` (default false)
       - Audit logging must be [enabled for your MongoDB Atlas project](https://www.mongodb.com/docs/atlas/database-auditing/) in order to scrape audit logs.
+    - `access_logs`
+      - `enabled` (default true, if the `access_logs` parameter is defined)
```

Comment:
This also leads us into questioning whether these should be on by default if a user is getting logs. I feel like we should maybe disable by default at first as to not increase costs of current users and then follow up with enabling it? 

@djaglowski @dehaansa do you guys have any thoughts?

Reply:
My comment should be clearer. Access Log are only "enabled" by default if you provide at least `access_logs:`, so they are realistically disabled by default but that could be made much more clear.

As far as the `enabled` pattern, I also do not like it but was trying to make MongoDBAtlas internally consistent since there are so many different potential components.

Reply:
> @djaglowski correct me if I'm wrong but I think we are trying to avoid the `enabled` portions of configs with new functionality and rather opt for existence of fields as to whether they are enabled? I.e trying to reduce a redundant parameter flow.

There's been some recent discussion on this and I was wrong to make this assumption as others disagree. I think it's probably best to leave it, especially since we're using it elsewhere in this receiver.

Reply:
> This also leads us into questioning whether these should be on by default if a user is getting logs. I feel like we should maybe disable by default at first as to not increase costs of current users and then follow up with enabling it?
> 
> @djaglowski @dehaansa do you guys have any thoughts?

I agree with leaving it disabled by default, to protect current users. As to whether that should change in the future - probably not unless users are asking for it.